internal/services/apimanagement/api_management_policy_fragment_resource.go

```diff
@@ -0,0 +1,232 @@
+	"github.com/hashicorp/terraform-provider-azurerm/internal/timeouts"
+)
+
+func resourceApiManagementPolicyFragment() *pluginsdk.Resource {
+	return &pluginsdk.Resource{
+		Create: resourceApiManagementPolicyFragmentCreate,
+		Read:   resourceApiManagementPolicyFragmentRead,
+		Update: resourceApiManagementPolicyFragmentUpdate,
+		Delete: resourceApiManagementPolicyFragmentDelete,
+		Importer: pluginsdk.ImporterValidatingResourceIdThen(func(id string) error {
+			_, err := policyfragment.ParsePolicyFragmentIDInsensitively(id)
+			return err
+		}, func(ctx context.Context, d *pluginsdk.ResourceData, meta interface{}) ([]*pluginsdk.ResourceData, error) {
+			client := meta.(*clients.Client).ApiManagement.PolicyFragmentClient
+
+			id, err := policyfragment.ParsePolicyFragmentIDInsensitively(d.Id())
+			if err != nil {
+				return nil, err
+			}
+
+			resp, err := client.Get(ctx, *id, policyfragment.GetOperationOptions{
+				Format: pointer.To(policyfragment.PolicyFragmentContentFormatXml),
+			})
+			if err != nil {
+				if response.WasNotFound(resp.HttpResponse) {
+					return nil, fmt.Errorf("Api Management Policy Fragment %q was not found in Api Management instance %q in Resource Group %q", id.PolicyFragmentName, id.ServiceName, id.ResourceGroupName)
+				}
+
+				return nil, fmt.Errorf("retrieving Api Management Policy Fragment %q (Api Management: %q, Resource Group %q): %+v", id.PolicyFragmentName, id.ServiceName, id.ResourceGroupName, err)
+			}
+
+			d.Set("format", policyfragment.PolicyFragmentContentFormatXml)
+
+			return []*pluginsdk.ResourceData{d}, nil
+		}),
+
+		Timeouts: &pluginsdk.ResourceTimeout{
+			Create: pluginsdk.DefaultTimeout(30 * time.Minute),
+			Read:   pluginsdk.DefaultTimeout(5 * time.Minute),
+			Update: pluginsdk.DefaultTimeout(30 * time.Minute),
+			Delete: pluginsdk.DefaultTimeout(30 * time.Minute),
+		},
+
+		Schema: map[string]*pluginsdk.Schema{
+			"name": schemaz.SchemaApiManagementChildName(),
+
+			"resource_group_name": commonschema.ResourceGroupName(),
+
+			"api_management_name": schemaz.SchemaApiManagementName(),
+
+			"format": {
+				Type:     pluginsdk.TypeString,
+				ForceNew: true,
+				Optional: true,
+				ValidateFunc: validation.StringInSlice([]string{
+					string(policyfragment.PolicyFragmentContentFormatRawxml),
+					string(policyfragment.PolicyFragmentContentFormatXml),
+				}, false),
+				Default: policyfragment.PolicyFragmentContentFormatXml,
+			},
+
+			"value": {
+				Type:             pluginsdk.TypeString,
+				Required:         true,
+				DiffSuppressFunc: XmlWithDotNetInterpolationsDiffSuppress,
+			},
+
+			"description": {
+				Type:     pluginsdk.TypeString,
+				Optional: true,
+			},
+		},
+	}
+}
+
+func resourceApiManagementPolicyFragmentCreate(d *pluginsdk.ResourceData, meta interface{}) error {
+	client := meta.(*clients.Client).ApiManagement.PolicyFragmentClient
+	ctx, cancel := timeouts.ForCreateUpdate(meta.(*clients.Client).StopContext, d)
```

Comment:
(as below) since this is only for the Create method and isn't a combined Create/Update method can we update this:

```suggestion
	ctx, cancel := timeouts.ForCreate(meta.(*clients.Client).StopContext, d)
```